test/e2e/prometheus_test.go

```diff
@@ -228,7 +228,7 @@ func TestPrometheusRemoteWrite(t *testing.T) {
     tlsConfig:
       ca:
         secret:
-          name: %[2]s
+          name:%[2]s
```

Comment:
One last nit :) This change seems unintended and unrelated?

Reply:
No, that one actually is an improvement. When I was playing around with the tests I noticed that the ConfigMap was not properly formatted when I printed it with `-o yaml` with "\t" and "\n" showing  and it was because of this space.